tests/integration/utility_test_functions.py

```diff
@@ -32,3 +36,86 @@ def run_cromshell_command(command: list, exit_code: int):
             f"\n{print_exception(*result.exc_info)}"
         )
         return result
+
+
+def wait_for_workflow_completion(test_workflow_id: str):
+    import time
+
+    count = 0
+    status = ""
+    print("Printing workflow status:")
+    while count < 10:
+        time.sleep(5)
+        count += 1
+        # run status check every 5 sec until done, max 10x
+        status_result = run_cromshell_command(
+            command=["status", test_workflow_id],
+            exit_code=0,
+        )
+        print(status_result.stdout)
+        status_result_formatted = json.loads(status_result.stdout)
+        status = status_result_formatted["status"]
+
+        if status == "Failed" or status == "Succeeded":
+            break
+
+    assert status == "Failed" or "Succeeded", "Workflow did not complete"
+
+
+def submit_workflow(
+    local_cromwell_url: str,
```

Comment:
Q: Are all automated tests currently run for local Cromwell?
Is it possible to mock one that's not local?
(unrelated to this PR)

Reply:
yes, all tests are local. A non local cromwell server test would require some form of authentication to be implemented to allow cromshell to connect to the external server.